sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -767,13 +654,8 @@ class ProximityStump(BaseClassifier):
     Parameters
     ----------
     random_state: integer, the random state
-    get_exemplars: function
-        extract exemplars from a dataframe and class value list
-    setup_distance_measure: function
-        setup the distance measure getters from dataframe and class value list
-    get_distance_measure: distance measure getters
     distance_measure: distance measures
-    get_gain: function to score the quality of a split
+    gain_name: function name to score the quality of a split, if None, gini_gain is used
```

Comment:
is "gini_gain" the only option? If so, I'd remove this entirely and make a comment note for developers how to add more functions instead.

Reply:
going to remove it and use `gini_gain` as a gain function throughout the classes and no need to include it in object instantiation. Just added a comment explaining the addition of more gain functions